extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -157,15 +159,21 @@ The `services` property contains an array of service configurations. There must
 If you'd like to test the document's behavior in the context of a particular viewer, you can add `#viewerUrl=` fragment parameter. For instance, `#viewerUrl=https://www.google.com` would emulate a document's behavior inside a Google viewer.
 
 
-## Selecting platform
-So if no platforms are selected, we compete all the platforms based on platforms like
+## Selecting a Service
+If no service returns an entitlement that grants access. All services are compared by calculating a score for each and the highest scoring service is selected.
```

Comment:
I misunderstood this one, change "." to ",":

If no service returns an entitlement that grants access, all services are compared by calculating a score for each and the highest scoring service is selected.

Reply:
got it, will push a fix with next bash of code changes.